### DISCUSSION TITLE
Welcome to Versatile Thermostat Discussions!

### DISCUSSION THREAD

**Discussion by jmcollin78** [jmcollin78/versatile_thermostat#151] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ba-pt0u** [comment]
Hello Hello,

I'm french, I started home automation few years ago with Jeedom and recently i mirate everything to HA.
Migration possible thanks to You @jmcollin78 because _Versatile Thermostat_ is the one who seems to be almost as good as Thermostat on Jeedom. 
My home heated by Electrical heater as main source and wood-burner as secondary heater (more for the pleasure of the fire).

Baptiste

**Comment by jmcollin78** [comment]
Merci Baptiste, ca me fait super plaisir !

**Comment by EPicLURcher** [comment]
Hi,

I live in the UK and use a Gas boiler for water radiators, (each radiator has a TRV valve on, which can adjust to temperature control an individual room). I also have electric A/C for the summer months, but do not have a "smart" way to  control these as of yet!

I've been using this integration for a while now and it is fantastic!

If you want help to review / update any of the English translations, let me know :)

**Reply by jmcollin78** [reply]
Ooooh yes my english is so poor. I would to have a real review for the README and the translations (in the custom_components/versatile_thermostat/translations/en.json.

I would love to have your review.

**Reply by EPicLURcher** [reply]
I can do that in the next few days :)

Do you want me to grab some en screenshots too?

**Reply by jmcollin78** [reply]
Yes it could be fine. Mostly for the different configuration steps.
Can you do that in a PR directly ?

**Reply by rpscholtz** [reply]
I there @EPicLURcher im looking into using this, have 2 x Sonoff TRV's to experiment with a Hive smart thermostat for radiators and for hot water any tips on how to configure the integration?

**Reply by Krzysztonek** [reply]
> any tips on how to configure the integration?

I'd recommend to start from [here](https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/en/creation.md).
Entire documentation you can find [here](https://github.com/jmcollin78/versatile_thermostat#documentation).

**Reply by rpscholtz** [reply]
Im still struggling to understand the setup - a bit too complicated - are there any video's of instructions or example setups I can look at?

**Comment by Kriss1670** [comment]
Bonjour.
Encore merci pour cette intégration !
Une petite coquille dans le texte de l’aide :
![IMG_8871](https://github.com/jmcollin78/versatile_thermostat/assets/80743001/526defc5-7e88-47da-8fcc-65a5ab2b7d15)
Je suppose que tu parles de « Versatile Thermostat » et pas de « Awesome thermostat »… 😉

**Reply by jmcollin78** [reply]
Au moins on sait d'où ça vient. Merci pour le signalement.

**Comment by Manulemalin64** [comment]
Bonjour,
J’utilise  actuellement Better thermostat qui m’a déjà grandement faciliter la vie, car auparavant, j’avais des têtes Figaro .J’avais paramétré toutes les fonctionnalités manuellement (Température, température eco, confort, etc… Gestion des absences des différentes personnes grâce à l’application Home assistant pour chaque pièce…)
Je teste donc actuellement, versatile thermostat qui regroupe tous les paramètrages que j’avais fait manuellement (Ça fonctionne bien, mais tout est regroupé sur votre application, Ça fait moins usine à gaz)
Une petite suggestion qui pourrait être appréciable, serait la possibilité d’ajouter un capteur d’humidité puisque vous proposez d’utiliser la carte UI de Better thermostat où l’on peut afficher le pourcentage d’humidité.
En tout cas, bravo pour cette application qui va Simplifier la mise en place d’un système de chauffage sur Home Assistant

Emmanuel

**Reply by jmcollin78** [reply]
Thank you @Manulemalin64 !

Adding the humidity is already planned: https://github.com/jmcollin78/versatile_thermostat/issues/167
Because it doesn't have a great added value, I postponed that improvement for later but it is not forgotten

**Reply by Manulemalin64** [reply]
Hello,

Thank you for this answer. Even if it's postponed, it's good news

Effectively the addition of humidity and less important 😉. But this adds a little extra, because sometimes we understand better why the feeling of cold is stronger (because of humidity) while the target temperature is reached

**Comment by gege-fr** [comment]
Hello there from France also ;-)

I'm new to HA and i'm installing it mainly to benefit from Versatile Thermostat.

I intend to install 14 TRVs on all my heaters. I'm waiting for the plumber to install the thermostatic valve body. I intend to order the SONOFF TRVs, is it a good bet ? I've seen that you've done quite and amount of work to support them. Can it be considered as reliable or is there a better choice ?

Gwenhaël

**Reply by jmcollin78** [reply]
Hello @gege-fr ,

Sonoff TRVZB is the best choice. You will have the best possible regulation with a the 0.1 °C (dépending of your configuration).
You have a affiliate link here if you want to order and help me to maintain the solution: https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/fr/presentation.md#materiels

([⭐ 4 x Sonoff TRVZB Zigbee](https://www.domadoo.fr/fr/chauffage-connecte/7478-sonoff-pack-de-4x-tete-thermostatique-connectee-zigbee-30.html?domid=97))

**Reply by jmcollin78** [reply]
You have also a full post with the installation and configuration of Sonoff TRVZB with Versatile Thermostat here: https://www.hacf.fr/tete-thermo-sonoff-trvzb/

**Reply by gege-fr** [reply]
Thanks, i'll have a look.

Is it "mandatory" to have an external temperature sensor in each room, or is it enough to rely on the TRVZB's internal sensor for bedrooms or toilets for example ?

**Reply by jmcollin78** [reply]
Yes it is mandatory. Internal temp sensor are mostly wrong because they are too close of the heating part. You can always create a sensor from the internal sensor of the TRVZB but the result in terms of regulation will be bad.

**Reply by gege-fr** [reply]
Okay, thanks, i'll take some for the most important rooms and i'll see how it goes for other rooms.

**Comment by Malik-h78** [comment]
Bonjour à tous,

Nouveau sur GitHub et de manière plus générale dans la domotique, je voudrais vous remercier pour les divers sujets que j'ai pu lire et grâce auxquels j'ai pu apprendre un peu plus.

Je suis actuellement en cours de rénovation d'une maison, et je souhaiterais pouvoir piloter le chauffage (alimenté par une pompe à chaleur air-eau deDietrich) et les têtes thermostatiques via Home assistant. Quelqu'un ici a déjà fait cela? J'ai l'impression qu'aucun thermostat connecté n'est compatible avec ma PAC à part celui de DeDietrich

Merci par avance pour vos retours d'experience

**Reply by jmcollin78** [reply]
Hello, ce qui est important c'est que ta PAC soit commandable via Home assistant a travers une entité de type `climate`.
Pour les têtes thermostatiques, l'idéal est que HA puisse commander directement la vanne comme c'est le cas avec les SONOFF TRVZB. Ils n'ont pas besoin d'être compatibles avec ta PAC puisque c'est Versatile Thermostat qui fait le lien entre les 2.

**Reply by Malik-h78** [reply]
Merci pour le retour, 

j'ai lu sur certains forums qu'avec l'application Remeha (qui est le duplicata de l'application deDietrich que j'utilise actuellement pour piloter le chauffage) il était possible d'intégrer mon thermostat dans HA avec cette intégration : 

https://github.com/msvisser/remeha_home

Remeha fonctionne avec mes identifiants deDietrich, ce qui signifie qu'une fois l'intégration faite, je pourrais piloter le chauffage via HA si je comprends bien. 

J'ai tout de même une question dans le cas où je souhaiterais atteindre une certaine température dans une chambre par exemple alors que le thermostat principal a atteint la température de consigne dans le salon. Comment ce genre de cas est géré?

Désolé pour les questions peut être un peu bêtes, mais je démarre tout juste et voudrais être sûr de pouvoir faire ce que je souhaite avant d'investir (dans une box domotique, dès tête thermostatiques, etc etc)

**Reply by jmcollin78** [reply]
Si tu as un TRV (tête thermostatique) dans la chambre, VTherm va la controler et l'ouvrir / fermer jusqu'à ce que la température de consigne soit atteinte. Il faut aussi un thermomètre connecté dans la pièce placé là où la température sera mesurée.

**Reply by Malik-h78** [reply]
Mais si la température de consigne est atteinte dans la pièce où est le thermostat principal qui contrôle la PAC mais que la température de consigne de la TRV n'est pas atteinte dans la chambre, la tête thermostatiques peut prendre le dessus sur le thermostat principal et "ordonner" a la PaC de chauffer?

**Reply by jmcollin78** [reply]
Oui le principe est que si un Vtherm demande du chauffage, la chaudière centrale est controlée (allumée / temperature changée / ...). Essaye de lire un peu la doc du coup : https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/fr/feature-central-boiler.md

C'est la fonction "chaudière centrale" qui fait ça. Tu peux choisir les commandes a envoyer à la chaudière lorsque la chauffe est nécessaire et la commande a envoyer lorsque ce n'est plus nécessaire.

**Reply by Malik-h78** [reply]
Merci, j'avais déjà lu ce passage cet après midi et bien compris que ce serait sûrement la fonction à utiliser dans mon cas. Mais le language encore un peu technique pour moi me mettais un doute, notamment sur le fait que mon appli ne gère pas du on/off sur la chaudière mais seulement un on lorsque la température est sous la consigne. Probablement que je pourrais une fois configuré dans HA.

En tout cas ces échanges me rassurent sur le fait que c'est faisable, et que je devrais pouvoir faire ce que je souhaite avec un peu de temps à passer sur HA. Et vu que je souhaite aussi piloter de l'éclairage et des volets roulants, je pense que l'investissement dans une box domotique ainsi que les têtes thermostatiques connectées vaut le coup (d'ailleurs si tu as des conseils sur la box domotique, ce n'est pas de refus, pour les têtes je prendrai très sûrement celles que tu conseilles, les sonoff en passant par ton lien)

En tout cas merci pour les conseils et la disponibilité c'est top et ne manquerai pas de te dire ce que j'ai réussi à faire et qui pourra peut être aussi aider quelqu'un qui est dans la même config et à les même doutes que moi.

**Reply by Malik-h78** [reply]
Je viens de voir cette partie qui m'avait échappé : 

<img width="578" height="210" alt="image" src="https://github.com/user-attachments/assets/5bc91344-42ee-40be-99c3-b05a3617f16e" />

donc même si l'appli de ma chaudière ne gère pas du "ON/OFF" a proprement dit, il suffirait de contourner en mettant des TRV sur tous les radiateurs et c'est eux qui donneraient les consignes, et donneraient l'info au thermostat relié à la chaudière de monter la consigne à 25° par exemple pour qu'elle se déclenche. Une fois la température atteinte sur tous les TRV, la consigne serait envoyée au thermostat de baisser à 10° pour arrêter la PAC

c'est un peu plus clair pour moi maintenant 👍

**Reply by gege-fr** [reply]
Bonjour ici,

Pour info c'est en effet +/- ce que j'ai l'intention de faire (j'attends encore le passage du plombier pour mettre les têtes thermostatiques).

En ce qui me concerne ma PAC elle n'a pas de thermostat d'ambiance, elle fonctionne en loi d'eau, il y a un bypass pour éviter qu'il y aie trop de pression sur la pompe et dans les circuits si tous les radiateurs sont fermés.

Peut être que ça peut aussi être votre cas sinon pour éviter de bidouiller le thermostat d'ambiance, le désactiver et repasser en loi d'eau.

**Reply by Malik-h78** [reply]
Bonjour,

je pensais que la loi d'eau était la courbe de chauffe de l'eau des radiateurs en fonction de la température extérieure. Mais celle ci était indépendante du thermostat d'ambiance qui lui s'assure de la bonne température a l'intérieur du domicile...

j'ai peut être loupé quelque chose

**Reply by gege-fr** [reply]
C'est bien le cas elle est indépendante de la température intérieure.

Elle permet cependant après réglages un température "grosso modo" a la consigne.

Mais bientôt ce seront les thermostats qui s'en assureront.

La loi d'eau servira uniquement a choisir la température de l'eau qui circule. Et les thermostats en choisiront le débit. Je remonterais peut être un peu ma loi d'eau pour que les thermostats ne peinent pas trop a monter la température lorsqu'ils s'ouvrent a fond.

**Reply by gege-fr** [reply]
Il me semble que dans certains cas (par exemple les TA de ma PAC (viessmann)), la loi d'eau est changée dynamiquement par le TA en fonction de la température souhaitée et de la température captée par le TA.

**Reply by jmcollin78** [reply]
> d'ailleurs si tu as des conseils sur la box domotique, ce n'est pas de refus, pour les têtes je prendrai très sûrement celles que tu conseilles, les sonoff en passant par ton lien)

Pour pas se prendre la tête et pour les petites config, il n'y a pas mieux que ça: https://www.hacf.fr/migrer-vers-ha-green-box/

Pour les TRV je conseille ce qui est indiqué ici (avec des liens affiliés): https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/fr/presentation.md#materiels

Je conseille les Sonoff TRVZB (VTherm a déjà développé avec ce modèle qui est donc totalement compatible maintenant). Tu as d'autres conseils matériels aussi.

**Reply by jmcollin78** [reply]
J'ai aussi fait un article complet pour installer et configurer les Sonoff TRVZB: https://www.hacf.fr/tete-thermo-sonoff-trvzb/

**Reply by gege-fr** [reply]
Je vais me retrouver avec 14 trvzb, il y a des conseils pour s'organiser ?

J'ai l'intention de les nommer au marqueur avec une lettre, de les nommer avec la même lettre dans HA, puis ensuite de les associer a un thermostat associé a une pièce donnée...

**Reply by Malik-h78** [reply]
Merci pour tous les conseils @jmcollin78, je regardais déjà vers la home assistant Green après divers lectures et pour ce que je vais en faire pour bien démarrer.

Maintenant y'a plus qu'à commander et commencer à bosser!

Je vous tiendrais au courant une fois tout installé ;)

**Reply by gege-fr** [reply]
Bonjour ici, 

J'ai partiellement finit mon installation, 

11 des 14 trvzb sont installés ça marche plutôt très bien, merci pour ça déjà !

J'ai une question, quand je fais un feu on dirait que ça perturbe l'algo. Et il "sous-chauffe" par la suite peut être car il comprend pas qu'il y a une source de chaleur externe.

Il y aurait des paramètres a ajuster pour limiter ce phénomène ?

Ci joint la courbe de la nuit d'hier, la consigne est a 21.5, c'est un capteur de température tiers qui contrôle 3 trvzb.
![Screenshot_20251123_102204_Home Assistant](https://github.com/user-attachments/assets/0079f4eb-0a95-4d16-8869-e24e590c9a71)

**Reply by gege-fr** [reply]
Et avec un peu plus de recul sur le comportement sur la journée ;-)

![Screenshot_20251123_211849_Home Assistant](https://github.com/user-attachments/assets/c46b7388-1932-43fb-b0f0-27b2a6d8a8ea)

**Comment by Kriss1670** [comment]
Bonjour Jean Marc,
Toujours fana de cette intégration très complète, merci pour le boulot !
Avant-hier, une sonde de température a cessé d'envoyer des données et le thermostat est passé en mode Safe. Je ne l'ai détecté que tardivement. Serait-il possible d'avoir une notification automatiquement lorsqu'un thermostat passe dans ce mode ou dois-je le faire via une automatisation ?
Merci beaucoup.

**Reply by jmcollin78** [reply]
Bonjour tout existe déjà, un p'tit tour dans la doc dans la section évènement et tu vas trouver.

**Comment by Kriss1670** [comment]
Merci Jean Marc, je sais le temps que tout ceci prend...
Je n'ai pas réussi à trouver le lien vers la section évènements. La seule chose qui s'en rapproche est la section Troubleshooting, partie "Comment être averti lorsque cela se produit qui contient bien un lien "Evènements" mais qui est inactif (rien ne s'ouvre).
J'ai créé une automatisation qui semble fonctionner :

alias: Alerte thermostat en sécurité
description: ""
triggers:
  - entity_id:
      - climate.thermostat_bureau
      - climate.thermostat_cuisine
      - climate.thermostat_sejour
      - climate.thermostat_wc_bas
    attribute: safety_state
    to: "on"
    trigger: state
conditions: []
actions:
  - data:
      title: ⚠️ Thermostat en sécurité
      message: >
        Le thermostat {{ state_attr(thermostat, 'friendly_name') }} est passé en
        mode sécurité. Vérifier la sonde de température
        correspondante.
    action: persistent_notification.create
variables:
  thermostat: "{{ trigger.entity_id }}"
mode: parallel
max: 5
________________________________
De : Jean-Marc Collin ***@***.***>
Envoyé : samedi 11 octobre 2025 08:53
À : jmcollin78/versatile_thermostat ***@***.***>
Cc : Kriss1670 ***@***.***>; Comment ***@***.***>
Objet : Re: [jmcollin78/versatile_thermostat] Welcome to Versatile Thermostat Discussions! (Discussion #151)


Bonjour tout existe déjà, un p'tit tour dans la doc dans la section évènement et tu vas trouver.

**Reply by jmcollin78** [reply]
Je t'aide: https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/fr/reference.md#ev%C3%A8nements

**Comment by Kriss1670** [comment]
Dans "Reference"... merci et désolé du dérangement !
________________________________
De : Jean-Marc Collin ***@***.***>
Envoyé : samedi 11 octobre 2025 11:52
À : jmcollin78/versatile_thermostat ***@***.***>
Cc : Kriss1670 ***@***.***>; Comment ***@***.***>
Objet : Re: [jmcollin78/versatile_thermostat] Welcome to Versatile Thermostat Discussions! (Discussion #151)


Je t'aide: https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/fr/reference.md#ev%C3%A8nements

**Reply by jmcollin78** [reply]
Pas de soucis. J'ai corrigé le lien qui était cassé.

**Comment by Alorcaalors** [comment]
Bonjour, je m'appelle Pierre, je réside en Savoie, dans une maison de 1930 pas super bien isolée mais en cours de rénovation.
Mon moyen de chauffage principal est une pompe à chaleur Aermec de 20 kW sur nappe géothermique, alimentant un circuit hydraulique de radiateur en fonte ayant l'âge de la maison...
Je sais, ce n'est pas idéal mais comme je vous l'ai dit, c'est en cours de rénovation. 
Cette pompe à chaleur était pilotée par un thermostat filaire non communicant, je l'ai changé par un modèle amélioré pilotable Delta dore. 
Je viens de passer une étape en utilisant versatile thermostat et un réseau de capteurs de température qui me donne une valeur moyenne pour la maison.
Démarrage un petit peu laborieux mais au bout de trois jours, et avec l'aide précieuse de jmcollin78, j'ai enfin trouvé un mode de fonctionnement stable, qui garantit des cycles suffisamment longs pour ne pas mettre en péril ma pompe à chaleur et avec des courbes de chauffe que je vais peaufiner, bien évidemment, mais qui pour l'instant sont tout à fait acceptables.

**Comment by jnilo1** [comment]
Bonjour,
Je viens de terminer l'installation de Vesartile Thermostat dans ma résidence secondaire en connectant les 9 fils pilotes de mon plancher chauffant à 3 relais 4 voies, le tout piloté par Z2M+HA sous docker et en utilisant une passerelle Lidl [modifiée par mes soins](https://github.com/jnilo1/hacking-lidl-silvercrest-gateway).
Je suis impressionné par la puissance et la versatilité... de VS. Un grand merci à @jmcollin78 pour ce développement !
J'ai quand même une question sur auto TPI que j'ai eu un peu de mal à mettre en place mais je vais la poser sur la section ad-hoc du forum (et in english of course)...
NB: ma question est [là](https://github.com/jmcollin78/versatile_thermostat/discussions/1840)
